ui/appui-react/src/appui-react/layout/widget/widget-content-layout/header/HeaderFirstRow.tsx

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import "./HeaderFirstRow.scss";
+import classNames from "classnames";
+import React from "react";
+
+import { HeaderTopRight } from "./HeaderTopRight.js";
+import type { IconMenu, IconMenuDivider, IconMenuSearch } from "./types.js";
+
+interface HeaderFirstRowProps {
+  /**
+   * CSS class name for the top left container.
+   */
+  topLeftClassName?: string;
+  /**
+   * Array of icons to display in the top right of the header.
+   */
+  icons?: IconMenu[];
+  /**
+   * Callback function to handle search input changes.
+   * Should be memoized to avoid unnecessary re-renders.
+   * @param {string} value - The new search value.
+   */
+  onSearch?: (value: string) => void;
+  /**
+   * Size of the icons in the header.
+   */
+  iconSize?: "small" | "large";
+  leftContent: React.JSX.Element[];
```

Comment:
`topLeftClassName` and `leftContent` naming is not consistent. Can we think of more descriptive name than `leftContent`?